tutor4points/core/templates/requests.html

```diff
@@ -14,14 +14,26 @@
 
 {% block content %}
 
+<div class="show-accepted">
+    <form method="post">
+    {% csrf_token %}
+        <div class="form-check">
+            <input class="form-check-input" type="checkbox" value="" id="show-accepted" name="show-accepted"
```

Comment:
Not a bug, but it would be nice to add styling to the checkbox so it matches the theme.